### PR TITLE
no longer being able to join bad code names

### PR DESCRIPTION
**Pull request by fchb1239**
- Game the colour page some more flare
- Fixed colours not being able to set
- Fixed where you couldn't group join mountains
- Fixed being able to join bad room code names

### CODE REVIEW

**Comment by AHauntedArmy** [comment]
i noticed setname also uses the wrong paremeters for initialize noob material, we should probably check every instance where that is called and update it.

File: ComputerInterface/BaseGameInterface.cs [hunk 1]
```diff
@@ -49,21 +46,36 @@ public static void SetName(string name)
...
+                InitializeNoobMaterial(r, g, b);
             }
             /* Player's name is not updating on change */
         }
 
+        public static void InitializeNoobMaterial(Color colour) => _InitializeNoobMaterial(colour);
+
+        public static void InitializeNoobMaterial(float r, float g, float b) => _InitializeNoobMaterial(new Color(r, g, b));
+
+        private static void _InitializeNoobMaterial(Color colour)
```
Comment (Graicc) [inline ComputerInterface/BaseGameInterface.cs]: Why is this hidden as a private method when InitializeNoobMaterial(Color colour)  is exposed

File: ComputerInterface/ComputerInterfaceCallbacks.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+using System.Text;
+
+using UnityEngine;
+
+using Photon.Pun;
+using Photon.Realtime;
+
+namespace ComputerInterface
+{
+    class ComputerInterfaceCallbacks : MonoBehaviourPunCallbacks
```
Comment (Graicc) [inline ComputerInterface/ComputerInterfaceCallbacks.cs]: Are these used anywhere?

Comment (fchb1239) [reply ComputerInterface/ComputerInterfaceCallbacks.cs]: They were for something that I deleted, I thought I deleted this too

**Pull request merged**